plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2613,7 +2613,7 @@ else if (hiveTableHandle.isInAcidTransaction()) {
                                 .map(hiveTypeMap::get)
                                 .collect(toImmutableList()),
                         OptionalInt.of(bucketProperty.get().getBucketCount()),
-                        !partitionedBy.isEmpty()),
+                        !partitionedBy.isEmpty() && isParallelPartitionedBucketedInsert(session)),
```

Comment:
parallel-partitioned-bucketed-inserts now applies to CTAS too.
This is a good change, but the property should be renamed, maybe "inserts" => "writes"

use `@LegacyConfig` and mind to update hive.rst if you do the change

Reply:
ok, will do